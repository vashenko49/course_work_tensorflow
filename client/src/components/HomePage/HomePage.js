import React, { Component } from 'react';
import { Container } from '@material-ui/core';
import Typography from '@material-ui/core/Typography';
import StyledLink from '../styled/StyledLink';
export class HomePage extends Component {
  render() {
    return (
      <Container>
        <Typography variant={'h3'}>
          Analysis of fluorograms for tuberculosis. Not a commercial version.
        </Typography>
        <StyledLink to={'analysis'}>
          <Typography variant={'body1'}>Analysis your fluorography -></Typography>
        </StyledLink>
      </Container>
    );
  }
}

export default HomePage;
