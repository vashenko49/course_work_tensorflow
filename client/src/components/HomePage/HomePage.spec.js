import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import { HomePage } from './HomePage';

describe('<HomePage />', () => {
  let shallow;
  beforeAll(() => {
    shallow = createShallow();
  });

  it('should work', () => {
    const wrapper = shallow(<HomePage />);
    expect(wrapper).toMatchSnapshot();
  });
});
