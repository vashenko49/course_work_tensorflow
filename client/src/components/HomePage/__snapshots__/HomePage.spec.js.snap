// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> should work 1`] = `
<WithStyles(ForwardRef(Container))>
  <WithStyles(ForwardRef(Typography))
    variant="h3"
  >
    Analysis of fluorograms for tuberculosis. Not a commercial version.
  </WithStyles(ForwardRef(Typography))>
  <_default
    to="analysis"
  >
    <WithStyles(ForwardRef(Typography))
      variant="body1"
    >
      Analysis your fluorography -&gt;
    </WithStyles(ForwardRef(Typography))>
  </_default>
</WithStyles(ForwardRef(Container))>
`;
